waisntechtools/alerts/messenger.py

```diff
@@ -0,0 +1,31 @@
+import requests
+from django.conf import settings
+from twilio.rest import Client
+from twilio.request_validator import RequestValidator
+from .get_twilio_cred import get_twilio_cred
+
+
+class Messenger(object):
+    def __init__(self):
+        self.client = get_twilio_cred()
```

Comment:
while the `get_twilio_cred()` function does handle the test environment scenario, it'll probably be more object oriented to allow this to be passed in through the constructor. It can serve as a default value though:

```
def __init__(self, client=get_twilio_client()):
        self._client = client
```

Two other things:
1. the function says it gets credentials which is assigned to a client variable name. This is confusing.
2. make the member variable private i.e. `self._client`